grep.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Grep
+category: CLI
+layout: 2017/sheet
+updated: 2019-10-04
+---
+
+### Usage
+{: .-prime}
+
+```bash
+grep <options> pattern <file...>
+```
+
+### Matching Options
+
+```bash
+-e PATTERN, --regexp=PATTERN
+-f FILE, --file=FILE
+-i, --ignore-case
+-v, --invert-match
+-w, --word-regexp
+-x, --line-regexp
+```
+
+### Pattern Options
+
+```bash
+-E, --extended-regexp # extended regular expression
```

Comment:
maybe add a brief summary of how ERE, BRE and PCRE differ, something like https://indigo.re/posts/2020-11-11-cli-regex.html#pattern-types

and maybe reorder from least powerful to most powerful: `-F`, `-G`, `-E`, `-P`